packages/@ama-sdk/schematics/schematics/api-extension/templates/_.gitignore

```diff
@@ -0,0 +1,39 @@
+# IDE project files
```

Comment:
could be renamed to `.gitignore.template` to avoid removing the `_`.
(same comment for the other files)